### DISCUSSION TITLE
About video time bar

### DISCUSSION THREAD

**Discussion by shiuyo** [mifi/lossless-cut#1285] [Q&A]
If we expand the video time ratio,for instance 2X 4X. How to select or locate time or peroid of invisible part behind. We can't see a progress bar at the bottom.

![Snipaste_2022-08-29_09-40-40](https://user-images.githubusercontent.com/112359978/187106494-1700ce56-805b-4e60-ba1b-52d0fb9c0449.png)

**Comment by mifi** [comment]
You mean a scroll bar? See #532
